src/Main.elm

```diff
@@ -28,25 +32,109 @@ init =
     { display = "0"
     , displayStatus = ShowingResults
     , decimalStatus = NoDecimal
+    , valueA = 0
+    , actionA = Blank
+    , valueB = 0
+    , actionB = Blank
     }
 
 
+type DisplayStatus
+    = ShowingResults
+    | NotResults
+
+
+type DecimalStatus
+    = YesDecimal
+    | NoDecimal
+
+
+orderOfOperations : Model -> Model
+orderOfOperations model =
+    case ( model.actionA, model.actionB ) of
+        ( Add, Equals ) ->
+            simpleMath model
+
+        ( Add, Add ) ->
+            simpleMath model
+
+        ( _, _ ) ->
+            model
+
+
+simpleMath : Model -> Model
+simpleMath model =
+    case model.actionA of
+        Add ->
+            { model
+                | display = String.fromFloat <| model.valueA + model.valueB
+                , valueA = 0
+                , actionA = Blank
+                , valueB = 0
+                , actionB = Blank
+                , displayStatus = ShowingResults
+            }
+
+        _ ->
+            model
+
+
 
 -- Update
 
 
 type Msg
     = UpdateDisplay CalButton
+    | Saveinput CalButton
 
 
-type DisplayStatus
-    = ShowingResults
-    | NotResults
+buttonToAction : CalButton -> Action
+buttonToAction button =
+    case button of
+        CalAdd ->
+            Add
 
+        CalEqual ->
+            Equals
+
+        _ ->
+            Blank
+
+
+calActionHit : Model -> CalButton -> Model
+calActionHit model button =
+    orderOfOperations <| saveAction model button
 
-type DecimalStatus
-    = YesDecimal
-    | NoDecimal
+
+saveAction : Model -> CalButton -> Model
+saveAction model button =
```

Comment:
The division of labor between `saveAction` and `orderOfOperations` with respect to checking the state of what's been entered so far (`valueA` vs `valueB`) feels strange to me. I think you could probably get away with having `valueB` at all if you combine the `case` logic between the two.

Reply:
I had this in here because I was thinking about what was coming. 
I thought I could handle all (order of operations) outcome as long as I knew 3 Values and 3 Actions (=,-,*,/) so that is why I had the 2nd value in there.
Looking at it now I do not think it was right since the commit was just to add Plus.
I will remove it and just leave what is needed for the story.